From 65130d13c59c13b7e5e07cfe69421ce1a08c0b7f Mon Sep 17 00:00:00 2001
From: Greg Hudson <ghudson@mit.edu>
Date: Tue, 17 Jul 2018 11:33:03 -0400
Subject: [PATCH] Remove outdated note in krb5kdc man page

Commit af5b77c887bfff24603715f8296c00d5eb839b0c (ticket 8348) removed
the interface-scanning workaround for platforms without pktinfo
support, so there is no longer an interaction between the krb5kdc -w
option and this workaround.

ticket: 8716 (new)
tags: pullup
target_version: 1.16-next

(cherry picked from commit 728b66ab867e31c4c338c6a6309d629d39a4ec3f)
---
 doc/admin/admin_commands/krb5kdc.rst | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/doc/admin/admin_commands/krb5kdc.rst b/doc/admin/admin_commands/krb5kdc.rst
index bda2c015c..b605b563d 100644
--- a/doc/admin/admin_commands/krb5kdc.rst
+++ b/doc/admin/admin_commands/krb5kdc.rst
@@ -72,13 +72,6 @@ will relay SIGHUP signals to the worker subprocesses, and will
 terminate the worker subprocess if the it is itself terminated or if
 any other worker process exits.
 
-.. note::
-
-          On operating systems which do not have *pktinfo* support,
-          using worker processes will prevent the KDC from listening
-          for UDP packets on network interfaces created after the KDC
-          starts.
-
 The **-x** *db_args* option specifies database-specific arguments.
 See :ref:`Database Options <dboptions>` in :ref:`kadmin(1)` for
 supported arguments.
